uWebshop.Payment.ePay/ePayPaymentBase.cs

```diff
@@ -1,10 +1,29 @@
-﻿namespace uWebshop.Payment.ePay
+﻿using System.Text;
+
+namespace uWebshop.Payment.ePay
 {
-	public class ePayPaymentBase
-	{
-		public string GetName()
-		{
-			return "ePay";
-		}
-	}
+    public class ePayPaymentBase
+    {
+        public string GetName()
+        {
+            return "ePay";
+        }
+
+        public static string MD5(string phrase)
+        {
+            var hasher = System.Security.Cryptography.MD5.Create();
+            var hashedDataBytes = hasher.ComputeHash(Encoding.UTF8.GetBytes(phrase));
+            return ByteArrayToString(hashedDataBytes);
+        }
+
+        private static string ByteArrayToString(byte[] inputArray)
+        {
+            var sb = new StringBuilder();
```

Comment:
you know beforehand the size of the inputArray you want to convert to string, so you know the size of the returning string (2*inputArray.Lenght). You initialize StringBuilder capacity with that value, you save one internal reallocation of it.